모의고사.js

```diff
@@ -0,0 +1,23 @@
+function solution(answers) {
+    const answer = [];
+    
+
+    let pik1 = [1,2,3,4,5];
+    let pik2 = [2,1,2,3,2,4,2,5];
+    let pik3 = [3,3,1,1,2,2,4,4,5,5];
```

Comment:
변하지 않는 값 => const 선언
변하는 값 => let 선언
줄바꿈 => 2줄이상 X, 문단별로 나누어 1줄 씩

```suggestion
    const answer = [];
    const pik1 = [1,2,3,4,5];
    const pik2 = [2,1,2,3,2,4,2,5];
    const pik3 = [3,3,1,1,2,2,4,4,5,5];
```